test/parallel/test-fs-readdir-recursive.js

```diff
@@ -1,14 +1,18 @@
 'use strict';
-const common = require('../common');
-const fs = require('fs');
-const net = require('net');
 
+const { test } = require('node:test');
+const fs = require('node:fs');
+const net = require('node:net');
+const common = require('../common');
```

Comment:
Common must be the first import. Also, I don't think we are accessing the `common` variable, so you can probably just do
```js
require("../common")

Reply:
We are using two things from `common`, I can destructure those and re-order to the top